tests/migration_test.go

```diff
@@ -2308,6 +2309,68 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 		})
 
 	})
+
+	Context("With Huge Pages", func() {
+		var hugepagesVmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			hugepagesVmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+		})
+
+		table.DescribeTable("should consume hugepages ", func(hugepageSize string, memory string) {
+			hugepageType := k8sv1.ResourceName(k8sv1.ResourceHugePagesPrefix + hugepageSize)
+			v, err := cluster.GetKubernetesVersion()
+			Expect(err).ShouldNot(HaveOccurred())
+			if strings.Contains(v, "1.16") {
+				hugepagesVmi.Annotations = map[string]string{
+					v1.MemfdMemoryBackend: "false",
+				}
+				log.DefaultLogger().Object(hugepagesVmi).Infof("Fall back to use hugepages source file. Libvirt in the 1.16 provider version doesn't support memfd as memory backend")
+			}
+
+			count := 0
+			nodes, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+			ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+			for _, node := range nodes.Items {
+				if v, ok := node.Status.Capacity[hugepageType]; ok && !v.IsZero() {
```

Comment:
might want to calculate that the node has more than enough huge pages to handle whatever the 'memory' var is.

Reply:
fixed